.github/workflows/build-linux-installer-rpm.yml

```diff
@@ -288,3 +288,70 @@ jobs:
     - name: Remove working files to exclude from cache
       run: |
         rm -rf ./chia-blockchain-gui/packages/gui/daemon
+
+  test:
+    name: Test package installation ${{ matrix.distribution.name }}
+    runs-on: ${{ matrix.os }}
+    needs:
+    - build
+    container: ${{ matrix.distribution.url }}
+    strategy:
+      fail-fast: false
+      max-parallel: 4
+      matrix:
+        os: [ubuntu-latest]
+        distribution:
+        - name: amazonlinux:2
+          type: amazon
+          url: "docker://amazonlinux:2"
+        - name: centos:7
+          type: centos
+          url: "docker://centos:7"
+        - name: fedora:34
+          type: fedora
+          # (34, 3.9) https://packages.fedoraproject.org/search?query=python3&releases=Fedora+34&start=0
+          url: "docker://fedora:34"
+        - name: fedora:35
+          type: fedora
+          # (35, 3.10) https://packages.fedoraproject.org/search?query=python3&releases=Fedora+35&start=0
+          url: "docker://fedora:35"
+        - name: fedora:36
+          type: fedora
+          # (36, 3.10) https://packages.fedoraproject.org/search?query=python3&releases=Fedora+36&start=0
+          url: "docker://fedora:36"
+        - name: rockylinux:8
+          type: rocky
+          url: "docker://rockylinux:8"
+        - name: rockylinux:9
+          type: rocky
+          url: "docker://rockylinux:9"
+        - name: build image
+          type: rocky
+          url: "docker://chianetwork/rocky8-builder:latest"
+        exclude:
+          - distribution:
+              name: amazonlinux:2
+          - distribution:
+              name: centos:7
+          - distribution:
+              name: rockylinux:8
+          - distribution:
+              name: build image
```

Comment:
These are the broken ones.  Amazon and CentOS are glibc version issues and rocky 8 and the (rocky 8 based) build image are dependency issues.